module-apps/application-call/windows/CallWindow.cpp

```diff
@@ -113,9 +113,9 @@ namespace gui
         sendSmsIcon->activatedCallback = [=](gui::Item &item) {
             LOG_INFO("Send message template and reject the call");
             constexpr auto preventAutoLock = true;
-            auto msg                       = std::make_unique<SMSSendTemplateRequest>(
-                presenter.getPhoneNumber().getView(), preventAutoLock, application->GetName());
+            auto msg = std::make_unique<SMSSendTemplateRequest>(presenter.getPhoneNumber().getView(), preventAutoLock);
             msg->ignoreCurrentWindowOnStack = true;
+            msg->nameOfSenderApp            = application->GetName();
```

Comment:
I'd rename it to `nameOfSenderApplication` as it's not a local variable, but a member variable and thus should have a name that could ease searching the code